lambdas/synthetic_monitors/cloudflare_sidekiq_worker.js

```diff
@@ -0,0 +1,87 @@
+// latency / "queue time" in seconds
+const thresholds = {
+  critical_external: 10,
+  critical: 10,
+  low: 43200
+}
+
+function thresholdExceeded(obj) {
+  let exceeded = false;
+  Object.keys(thresholds).forEach(threshold => {
+    if ( obj[threshold] >= thresholds[threshold] ) {
+      exceeded = true
+      return
+    }
+  })
+  return exceeded
+}
+
+function objectToString(obj) {
+  return Object.entries(obj)
+    .map(([key, value]) => `${key}: ${value}`)
+    .join('\n');
+}
+
+async function notify(text) {
+  return fetch("https://hooks.slack.com/services/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", {
+    method: "POST",
+    body: JSON.stringify({
+      text
+    }),
+    headers: {
+      "Content-type": "application/json; charset=UTF-8"
+    }
+  });
+}
+
+function validateResponseObject(obj) {
```

Comment:
I think you can rewrite the filteredObj construction so that it only does a single pass:

```
function validateResponseObject(obj) {
  const expectedProperties = Object.keys(thresholds);
  if (typeof obj !== 'object' || obj === null) {
    return { isValid: false };
  }

  const filteredObj = {};
  for (const key of expectedProperties) {
    if (obj[key] == null) return { isValid: false };
    filteredObj[key] = obj[key];
  }
  return { isValid: true, filteredObj };
}
```